StoreApp/Controllers/AccountController.cs

```diff
@@ -212,18 +213,21 @@ public ActionResult EditUserAdmin(UserViewModel user)
         {
             if ((Session["Role"] as string).Equals(UserRoles.admin.ToString()))
             {
-                user.Role = user.Role.Equals((int)UserRoles.user) ? UserRoles.user.ToString() 
-                                                                  : UserRoles.admin.ToString();
+                var i = UserRoles.user.ToString();
+                user.Role = int.Parse(user.Role) == (int)(UserRoles.user) ? UserRoles.user.ToString() 
+                                                                          : UserRoles.admin.ToString();
 
                 UserDTO userDTO = webMapper.config.Map<UserViewModel, UserDTO>(user);
 
                 userService.Edit(userDTO);
 
+                TempData["StatusMessage"] = "success";
                 TempData["Message"] = $"{user.UserName} edit success";
```

Comment:
message: **The UserName was updated successfully**
Check the messages below please

Reply:
fixed